graphql_compiler/tests/test_data_tools/data_tool.py

```diff
@@ -16,3 +36,68 @@ def generate_data(client):
                     continue
 
                 client.command(sanitized_command)
+
+
+@retry(tries=20, timeout=1)  # pylint: disable=no-value-for-parameter
+def init_sql_integration_test_backends():
+    """Connect to and open transaction on each SQL DB under test."""
+    sql_test_backends = {}
+    for backend_name, base_connection_string in six.iteritems(SQL_BACKEND_TO_CONNECTION_STRING):
+        engine = create_engine(base_connection_string)
+        # safely create the test DATABASE for all SQL backends except sqlite
+        # sqlite's in-memory database does not need to be explicitly created/dropped.
+        if backend_name in EXPLICIT_DB_BACKENDS:
+            # safely drop the test DB, outside of a transaction (autocommit)
+            safe_delete_command = text('DROP DATABASE IF EXISTS animals;')
+            engine.execution_options(isolation_level="AUTOCOMMIT").execute(safe_delete_command)
+            # create the test DB, outside of a transaction (autocommit)
+            create_command = text('CREATE DATABASE animals;')
+            engine.execution_options(isolation_level="AUTOCOMMIT").execute(create_command)
+            # update the connection string and engine to connect to this new DB specifically
+            connection_string = base_connection_string + u'/animals'
+            engine = create_engine(connection_string)
+        sql_test_backend = SqlTestBackend(engine, base_connection_string)
+        sql_test_backends[backend_name] = sql_test_backend
+    return sql_test_backends
+
+
+def tear_down_integration_test_backends(sql_test_backends):
+    """Rollback backends' transactions to wipe test data and to close the active connections."""
+    for backend_name, sql_test_backend in six.iteritems(sql_test_backends):
+        if backend_name not in EXPLICIT_DB_BACKENDS:
+            continue
+        # explicitly release engine resources, specifically to disconnect from active DB
+        # some backends including Postgres do no not allow an in use DB to be dropped
+        sql_test_backend.engine.dispose()
+        # connect to base server, not explicit DB, so DB can be dropped
+        engine = create_engine(sql_test_backend.base_connection_string)
+        # set execution options to AUTOCOMMIT so that the DB drop is not performed in a transaction
+        # as this is not allowed on some SQL backends
+        (engine.execution_options(isolation_level="AUTOCOMMIT")
```

Comment:
How about extracting the `text('...')` into a variable so that you can fit all of this on the same line and don't need the wrapping `()` around the entire thing?